code/game/objects/items/emags.dm

```diff
@@ -64,8 +65,21 @@
 	return ITEM_INTERACT_SUCCESS
 
 /obj/item/card/emag/ranged_interact_with_atom(atom/interacting_with, mob/living/user, list/modifiers)
+	if (can_emag(interacting_with, user))
+		if(pshoom_2 && prob(1))
+			playsound(src, pshoom_2, 40, TRUE)
+		else
+			playsound(src, pshoom, 40, TRUE)
+		user.Beam(interacting_with, icon_state = "rped_upgrade", time = 0.5 SECONDS)
 	return prox_check ? NONE : interact_with_atom(interacting_with, user)
 
+/obj/item/card/emag/item_interaction(mob/living/user, obj/item/tool)
```

Comment:
What's the return value of this? I'm noticing one branch returns NONE and another nothing? That's weird

Reply:
yeah you're right thats weird, forgot to add the return on this

Reply:
fixed it